db/models/flags_v2/flag_update.py

```diff
@@ -0,0 +1,38 @@
+from enum import IntEnum
+from uuid import uuid4
+
+from db import db
+from flask_sqlalchemy import DefaultMeta
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import String
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.sql import func
+
+BaseModel: DefaultMeta = db.Model
+
+
+class FlagStatus(IntEnum):
+    RAISED = 0
+    STOPPED = 1
+    RESOLVED = 3
+
```

Comment:
Do you need `ESCALATED` type here?

Reply:
Charles said to leave ascalated for now as they weren't sure quite what that meant or if they wanted to ahve it.